deeplab/scripts/convert_deeplab.sh

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env bash
+# =============================================================================
+# Copyright 2019 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+# Make sure that we run Python 3.6, not 3.7
+PYTHON_VERSION=$(python -V 2>&1 | sed 's/.* \([0-9]\).\([0-9]\).*/\1\2/')
+if [ "$PYTHON_VERSION" -gt "36" ]; then
+  echo "This script requires python 3.6 or older."
+  # exit 1
```

Comment:
Let's uncomment this.
```suggestion
  exit 1
```